plutus-metatheory/src/Type.lagda.md

```diff
@@ -81,7 +81,10 @@ scope.
 A type is either a type variable, a pi type, a function type, a
 lambda, an application, an iso-recursive type `μ`, a size, or a type
 constant (base type). Note that recursive types range over an
-arbitrary kind `k` which goes beyond standard iso-recursive types.
+arbitrary kind `k` which goes beyond standard iso-recursive types. Also note that `Π`,
```

Comment:
I always thought `Π` was for dependent types and what we have in Plutus is `∀`.

Reply:
You're right. But, \forall is reserved so I think Phil chose Pi for this.

Reply:
I suppose you could refer to it as "universal quantification `Π`" like we say "iso-recursive type `μ`"